lnbits/helpers.py

```diff
@@ -348,3 +351,33 @@ def normalize_path(path: Optional[str]) -> str:
 
 def normalized_path(request: Request) -> str:
     return "/" + "/".join(path_segments(request.url.path))
+
+
+async def send_email(
```

Comment:
this is a notification service, not a helper

Reply:
i would argue this is a smtp helper, you see it by the imports

Reply:
Id argue the same. I will use for extensions at some point.

Reply:
Nostr would be helpful as a helper as well, give it a privkey, pubkey and message.

Reply:
A helper something you call from crud, models, services, views or api.
A service is something you call from other services or from api.
`send_email` IS a service.